RELEASES.md

```diff
@@ -1,3 +1,105 @@
+Version 1.30.0 (2018-10-25)
+==========================
+
+Language
+--------
+- [This version marks the release of the 2018 edition of Rust.][54057]
+- [In the 2018 edition you can now use `crate` visibility keyword.][54057]
+  Equivalent to `pub(crate)`.
+- [You can now use keywords as identifiers using the raw identifiers
+  syntax (`r#`).][53236] e.g. `let r#struct = 2018;`
+- [Using anonymous parameters in traits is now deprecated with a warning and is
+  a hard error in the 2018 edition.][53272]
+- [You can now apply the `#[used]` attribute to static items to prevent the
+  compiler from optimising them away even if they appear to be unused.][51363]
+  e.g. `#[used] static FOO: u32 = 1;`
+- [You can now import and reexport macros using the `use` syntax.][50911]
+- [`mod.rs` files are now optional, you can now have `src/foo.rs` and
+  `src/foo/bar.rs` to define module dependencies.][54146]
```

Comment:
This sentence is really confusing to me. I'm not sure where `bar.rs` comes into play here, shouldn't we be comparing it to `src/foo/mod.rs`?